### ISSUE TITLE
Where place the folder ?

### ISSUE THREAD

**Issue by gautierd** [Craxy/ToggleTrafficLights#15]
Hello, I have a simply question it's where to place the folder because I put it in : 
C:/Program Files (x86)/Cities Skylines/Files/Mods 
but it doesn't work ...
Can you help us please ?
Thanks

**Comment by Craxy** [comment]
The folder for local mods is "%LOCALAPPDATA%\Colossal Order\Cities_Skylines\Addons\Mods\". See: http://docs.skylinesmodding.com/en/master/general/Folder-Structure.html

But unless you want to compile the code on your own or want a specific version, subscribe to the mod on the steam workshop -- it'll put the files automatically in the right places.

Edit: details for the "\Cities_Skylines\Files\Mods" folder: this folder contains the "build-in" mods -- ergo the official mods by CO, which are shipped with the game. They work as example-mods too.  

In a nutshell: 
- "[Steam]\SteamApps\common\Cities_Skylines\Files\Mods\": official mods by CO
- "%LOCALAPPDATA%\Colossal Order\Cities_Skylines\Addons\Mods\": mods, which were not obtained from the steam workshop
- "[Steam]\SteamApps\workshop\content\255710\": subscribed mods from the workshop

**Comment by gautierd** [comment]
Okay thank you for your answer but unfortunately it doesn't work in this folder LOCALAPPDATA%\Colossal Order\Cities_Skylines\Addons\Mods\

**Comment by Craxy** [comment]
Go to the [release page](https://github.com/Craxy/ToggleTrafficLights/releases), find your desired version of the mod and download "ToggleTrafficLights.zip". unzip it to "%LOCALAPPDATA%\Colossal Order\Cities_Skylines\Addons\Mods\". That creates a folder called ToggleTrafficLights" with the dll in it.  In game it should be shown under mods.

Again: Best subscribe to the mod on the steam workshop: that'll do everything for you AND updates automatically to new versions.

**Comment by Craxy** [comment]
No further response for the last 5 days.  
-> I assume (and hope) you found the correct folder and the mod works now.  
-> question closed
